scene/3d/bone_attachment_3d.cpp

```diff
@@ -148,7 +148,9 @@ void BoneAttachment3D::_check_bind() {
 			bone_idx = sk->find_bone(bone_name);
 		}
 		if (bone_idx != -1) {
-			sk->call_deferred(SNAME("connect"), "bone_pose_changed", callable_mp(this, &BoneAttachment3D::on_bone_pose_update));
+			if (!bound && !sk->is_connected(SNAME("bone_pose_changed"), callable_mp(this, &BoneAttachment3D::on_bone_pose_update))) {
```

Comment:
Isn't the whole point of the `bound` boolean to prevent having to do such checks of `is_connected`?

Is there a race condition here?

Reply:
I don't know. All I know is that the reimport tries to reconnect the same signals all over spamming the editor console with errors.

Reply:
We are already in a `!bound` if condition (L146). So technically, that check is duplicated/redundant.